src/main/java/org/skriptlang/skript/bukkit/loottables/LootTableModule.java

```diff
@@ -0,0 +1,125 @@
+package org.skriptlang.skript.bukkit.loottables;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.classes.Parser;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.lang.util.SimpleEvent;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.registrations.EventValues;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.world.LootGenerateEvent;
+import org.bukkit.loot.LootContext;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+
+public class LootTableModule {
+
+	public static void load() throws IOException {
+
+		// --- CLASSES --- //
+
+		Classes.registerClass(new ClassInfo<>(LootTable.class, "loottable")
+			.user("loot ?tables?")
+			.name("Loot Table")
+			.description("Loot tables represent what items should be in naturally generated containers, "
+					+ "what items should be dropped when killing a mob, or what items can be fished.",
+				"You can find more information about this in https://minecraft.wiki/w/Loot_table")
```

Comment:
```suggestion
			.description(
				"Loot tables represent what items should be in naturally generated containers, "
					+ "what items should be dropped when killing a mob, or what items can be fished.",
				"You can find more information about this in https://minecraft.wiki/w/Loot_table"
			)
```
Indentation a bit wonky